citesphere/src/main/java/edu/asu/diging/citesphere/core/zotero/impl/ZoteroManager.java

```diff
@@ -386,6 +491,22 @@ public ICitation createCitation(IUser user, String groupId, List<String> collect
         return itemTypeFields;
     }
 
+    private List<String> getNoteFieldList() {
+        List<String> noteFields = new ArrayList<>();
+        noteFields.add(ZoteroFields.ITEM_TYPE);
+        noteFields.add(ZoteroFields.PARENT_ITEM);
+        noteFields.add(ZoteroFields.TAGS);
+        noteFields.add(ZoteroFields.NOTE);
+        return noteFields;
+    }
+    
+    private boolean isMetaDataNote(Item item) {
```

Comment:
I believe there is a second method somewhere very simillar to this? I think I would add this method to the item class and then you can just call `item.isNote()` or something like that whenever you need it.